services-custom/s3-transfer-manager/src/test/java/software/amazon/awssdk/transfer/s3/progress/LoggingTransferListenerTest.java

```diff
@@ -19,11 +19,11 @@
 import static org.mockito.Mockito.mock;
 
 import java.util.List;
-import org.apache.logging.log4j.Level;
+import nl.altindag.log.LogCaptor;
 import org.apache.logging.log4j.core.LogEvent;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import software.amazon.awssdk.testutils.LogCaptor;
```

Comment:
Nice, I didn't know we have LogCaptor, can we use it instead?

Reply:
yep, using LogCaptor from testutils instead